src/Scaffold/Console/controller/controller.stub

```diff
@@ -8,14 +8,29 @@ use Backend\Classes\Controller;
  */
 class {{studly_name}} extends Controller
 {
+    /**
+     * @var array Extensions implemented by this controller.
```

Comment:
```suggestion
     * @var array Behaviors that are implemented by this controller.
```

Reply:
Not sure if this is good if core uses same descriptions. For example like here: https://github.com/octobercms/october/blob/439ed2cc33a1c4ec37a862d640ab7736fd193325/modules/backend/controllers/Users.php

Reply:
"Behaviors" is more accurate, we'll use it instead.